spec/javascript/containers/AirlineShowContainer.js

```diff
@@ -0,0 +1,48 @@
+import AirlineShowContainer from '../../../app/javascript/react/container/AirlineShowContainer';
+import AirlineShowTile  from '../../../app/javascript/react/components/AirlineShowTile'
+import { mount } from 'enzyme';
+import jasmineEnzyme from 'jasmine-enzyme';
+import React from 'react';
+import fetchMock from 'fetch-mock'
+
+describe('AirlineShowContainer', () => {
```

Comment:
Please fix indentation throughout file and remove/add empty lines (e.g., empty line should be added before `beforeEach`).

Reply:
noted -- thank you.